docs/documentation/sitespeed.io/configuration/config.md

```diff
@@ -94,6 +94,9 @@ WebPageTest
   --webpagetest.script             The WebPageTest script as a string.
   --webpagetest.includeRepeatView  Do repeat or single views  [boolean] [default: false]
   --webpagetest.private            Wanna keep the runs private or not  [boolean] [default: true]
+  --webpagetest.timeline           Activates chrome tracing. More Useful metrics but can slow down the test itself  [boolean] [default: false]
+  --webpagetest.domainsDashboard   Record weight and requests for each domain to populate the grafana dashboard. Will take more disk space in graphite [boolean] [default: false]
```

Comment:
We send over all webpagetest.* metrics to the WebPageTest API, I think maybe we should mark the ones that are sitespeed.io own, so it makes it easier to understand. @tobli @beenanner any input?

Reply:
I mean *webpagetest.domainsDashboard* :)

Reply:
Yeah I thought about deleting the option once it has been used in `index.js`, but modifying the options directly is never a good idea.

Later on, I would like to add a connection alias (rather than "custom") for WPT, so the same problem will occur.

Maybe we should add a sub namespace, like
```
--webpagetest.ssio.enableDomains=…
--webpagetest.ssio.connectivityAlias=…
```

Reply:
Yes, either that or adding *--webpagetestParams* (following the same patterns as we have for some others)? The thing with the WebPageTest NodeJS API is that it changes options passed into it but if I remember correctly we "fixed" that by passing a copy of the object to the api.

Thinking some more, if you add the .ssio and then we could remove it from the cloned version here: https://github.com/sitespeedio/sitespeed.io/blob/master/lib/plugins/webpagetest/analyzer.js#L23

Reply:
you're the architect, tell me what would fit best

Reply:
I want input from @beenanner and @tobli :)